### DISCUSSION TITLE
Unable to build Build/Core/Learner service release tag release-5.3.1_RC1

### DISCUSSION THREAD

**Discussion by srajasimman** [Sunbird-Lern/Community#178] [Installation] [answered]
Hi @reshmi-nair,

We are trying to build and deploy sunbird on our environment.
while building the service Learner service (Build/Core/Learner) we are facing some depenancy issues. we are looking for your support to proceed furter.

here is the error of build job
[Learner_build_log.txt](https://github.com/Sunbird-Lern/Community/files/13646879/Learner_build_log.txt)

Regards,
@srajasimman

cc: @Arunkumarpodili, @Priya-1123

**Comment by BharathwajShankar** [comment]
Hi @srajasimman, could you please share the steps you are trying to follow? In the local environment, when we try it, everything is working as expected. We can also connect over a call to check this further.

**Reply by srajasimman** [reply]
Hi @BharathwajShankar,

Thanks for responding my query. me (@srajasimman) and @Arunkumarpodili are available to connect with you. kindly let us know your availablity or send an meeting invite to rajasimman.s@optit.in and arunkumar.p@optit.in

Thanks and Regards,
Rajasimman S

**Reply by BharathwajShankar** [reply]
Have extended the meeting invite as requested.

**Reply by srajasimman** [reply, marked as answer]
Hi @BharathwajShankar,

Thank you so much for your support.

We found the issue was we are simultaneous that caused resource issue on the build jenkins VM.
when we tried to build one at a time the job got passed with success message.

Regards,
@srajasimman 

![image](https://github.com/Sunbird-Lern/Community/assets/15092596/f770904a-9133-4d15-99da-696d28b5627f)
